docs/contributing.rst

```diff
@@ -0,0 +1,203 @@
+.. _contrib_guide_reference-label:
+
+""""""""""""""""""
+Contributing Guide
+""""""""""""""""""
+
+We welcome contributions from external contributors, and this document describes how to merge code changes into
+:code:`toqito`. 
+
+================
+Getting Started
```

Comment:
I think the titles could be changed here. 

`Getting Started` has a section labeled `Contributing` and `Contributing Guide` also has a section labeled `Getting Started`.